### ISSUE TITLE
Strange number of results when varying "n" in "bh -n"

### ISSUE THREAD

**Issue by kootenpv** [rcaloras/bashhub-client#35]
Trying bh -n for some time:

```
pascal@MBP ~ $ bh -n 9999 > bhhist.txt
pascal@MBP ~ $ cat bhhist.txt | wc -l
      45
pascal@MBP ~ $
pascal@MBP ~ $ bh -n 2300 > bhhist.txt
pascal@MBP ~ $ cat bhhist.txt | wc -l
    2300
pascal@MBP ~ $ bh -n 5000 > bhhist.txt
pascal@MBP ~ $ cat bhhist.txt | wc -l
    1034
```

**Comment by rcaloras** [comment]
@kootenpv believe this was an encoding bug and is fixed in the 1.1.2 release by https://github.com/rcaloras/bashhub-client/commit/ac1a70f25a057874b5a3061fcbea6cd4d14f3c23.

run `bashhub update` and then try something like `bh -n 5000 | wc -l` you should at least get the number of lines represented by `-n`. If some commands are multiple lines, you could end up with more. 

Thanks for reporting. Let me know if it's fixed!

**Comment by kootenpv** [comment]
Fixed, fantastic, thanks a lot!
